seed/ts-sdk/seed.yml

```diff
@@ -31,6 +31,13 @@ fixtures:
         allowCustomFetcher: true
         timeoutInSeconds: 2
       outputFolder: allow-custom-fetcher
+  failing-audiences:
+    - customConfig: null
+      outputFolder: no-custom-config
+    - customConfig: null
+      audiences:
+        - partner
+      outputFolder: with-partner-audience
```

Comment:
is there a way to test these edge cases using the existing `audiences` fixture?

Reply:
yeah! let me add both like this!